src/openzaak/notifications/viewsets.py

```diff
@@ -36,3 +43,43 @@ def _message(self, data, instance=None):
                 )
 
                 transaction.on_commit(lambda msg=message: send_notification.delay(msg))
+
+
+type CloudEvenHandler = Callable[[CloudEvent], None | NoReturn]
+
+
+class CloudEventWebhook(APIView):
```

Comment:
This view currently doesn't require any auth and we also don't have any scopes (`vng_api_common.scopes.Scope`) defined for cloud events. The existing permission class `openzaak.utils.permissions.AuthRequired` doesn't seem to be appropriate for this because it requires a `main_resource`, which is not really applicable here.

Could you make sure a scope is added for publishing cloud events and that this scope is required for this view?